hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/state/migration/MigrationRecordsManager.java

```diff
@@ -154,7 +167,8 @@ private void publishAccountsCreated(
                 account.getMemo(),
                 memo,
                 description,
-                account.getBalance()));
+                account.getBalance(),
+                account.getAlias()));
```

Comment:
Why not check here if `getAlias()` returns `null` and if so pass in `ByteString.EMPTY` as at line 189 below? Then the `alias` parameter to `synthCreation` could be declared `@NonNull` which would be nicer (and simplify the test at L238 accordingly).